cirq/protocols/decompose_protocol.py

```diff
@@ -182,6 +181,9 @@ def decompose(
             returns `None`, non-decomposable operations are simply silently
             kept. `on_stuck_raise` defaults to a `ValueError` describing the
             unwanted non-decomposable operation.
+        preserve_structure: Prevents structural elements (i.e. subcircuits)
```

Comment:
```suggestion
        preserve_structure: Prevents subcircuits (i.e. `CircuitOperation`s)
```

Reply:
Applied change. My intent is for this to preserve other purely-structural circuit features as they emerge, but since that's only `CircuitOperation` for now I'm happy to limit the scope of the comment.